src/wakeflow/wakeflow.py

```diff
@@ -297,9 +311,9 @@ def _run_wakeflow(self, params: _Parameters) -> None:
             nonlin_perts = _NonLinearPerts(params, grid_nonlin_perts)
 
             # extract initial condition from the linear perturbations
-            nonlin_perts._extract_ICs(lin_perts)
+            nonlin_perts._extract_ICs_ann(lin_perts)
```

Comment:
I am confused about this change. Did you update `_extract_ICs_ann` so that it is now the preferred version? It was previously **not** the preferred version. The comment above the definition of it is also confusing since it says it IS the old version. I am sure this is all fine the current comments just don't seem compatible with what is actually happening.

Reply:
Sounds good.